src/game/game-jwt-auth.ts

```diff
@@ -0,0 +1,15 @@
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { GqlExecutionContext } from '@nestjs/graphql';
+import { Request } from 'express';
+
+/**
+ * Check if the user performing the request is authenticated.
+ */
+@Injectable()
+export class GameJwtAuthGuard extends AuthGuard('jwt') {
```

Comment:
Rename to `RestJwtGuard` and move to `common/guards`. It is something we might use in other (possibly) REST endpoints, right?